### PR TITLE
update discrimination paragraph

### PR DESCRIPTION
**Pull request by davharris**
(see #297 and #302)

### CODE REVIEW

**Comment by agitter** [comment]
I didn't review this but made a commit so that @davharris doesn't have to waste time learning the nuances of our citation system.  The build passes now.

@davharris you'll have to pull from your branch `davharris:patch-1` again if you want to make further changes.  I committed directly to your branch.

File: sections/03_categorize.md [hunk 1]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
-during a patient's visit to the doctor and there's an existing pattern of racial
-differences in prescription behavior (`TODO: @traversc - you can pick a
-different example but I think we need one - whichever one you think is most
-fully supported by the literature.`), this pattern could become baked into the
-predictions made by the algorithm. Machine learning practitioners, and
-particularly those who use deep neural networks, which are challenging to
-interpret, must remain cognizant of this possibility and make every effort to
-prevent harm from discriminatory predictions.
+important to consider algorithm interpretability alongside accuracy. A number of
+properties of genomic data will make this difficult.  First, research samples are
```
Comment (cgreene) [inline sections/03_categorize.md]: `genomic and health care` data?

File: sections/03_categorize.md [hunk 2]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
+in the general population than would be implied by case-control data, as reviewed 
+in [@doi:10.1086/512821]. Moreover, genetic associations that hold in one population 
+may not hold in other populations with different patterns of linkage 
+disequilibrium (even when population stratification is explicitly controlled for; 
+[@doi:10.1038/nrg2813]). As a result, many genomic findings are of limited value for 
+people of non-European ancestry[@doi:10.1371/journal.pbio.1001661] and may lead
+to worse treatment outcomes for them. Methods have been developed for mitigating 
+some of these problems in genomic studies [@doi:10.1086/512821 
+@doi:10.1038/nrg2813], but it is not clear how easily they can be adapted for 
+deep models that are designed specifically to extract subtle effects from
```
Comment (cgreene) [inline sections/03_categorize.md]: Good point!

File: sections/03_categorize.md [hunk 3]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
+(e.g. [@doi:10.1126/science.333.6041.404-a]); in some contexts, it may not be possible 
+to correct for all of these differences to the degree that a deep network is 
+unable to use them. Moreover, the complexity of deep networks makes it difficult
+to determine when their predictions are likely to be based on such 
+nominally-irrelevant features of the data (called "leakage" in other fields; 
+[@doi:10.1145/2382577.2382579]). When we are not careful with our data and models,
+we may inadvertently say more about the way the data was collected than about
+anything of scientific or predictive value, with potentially disastrous 
+and discriminatory consequences [@doi:10.1111/j.1740-9713.2016.00960.x]. 
+[@doi:10.1145/2382577.2382579] discuss some ways in which these problems can be
```
Comment (cgreene) [inline sections/03_categorize.md]: Can you make this: Kaufman et al. [@doi:...] discuss...

File: sections/03_categorize.md [hunk 4]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
+mitigated, but it remains difficult to predict when they will arise, how to
+diagnose them, and how to resolve them.
```
Comment (cgreene) [inline sections/03_categorize.md]: Is there any take-home recommendation that you'd have for those getting into deep learning in this space? I realize this is an open issue without a clear solution at this time.

File: sections/03_categorize.md [hunk 5]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
-example, if we train an algorithm to predict which drugs would be prescribed
-during a patient's visit to the doctor and there's an existing pattern of racial
-differences in prescription behavior (`TODO: @traversc - you can pick a
-different example but I think we need one - whichever one you think is most
-fully supported by the literature.`), this pattern could become baked into the
-predictions made by the algorithm. Machine learning practitioners, and
-particularly those who use deep neural networks, which are challenging to
-interpret, must remain cognizant of this possibility and make every effort to
-prevent harm from discriminatory predictions.
+important to consider algorithm interpretability alongside accuracy. A number of
```
Comment (agapow) [inline sections/03_categorize.md]: Eliminate "algorithm"

File: sections/03_categorize.md [hunk 6]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
+properties of genomic data will make this difficult.  First, research samples are 
+frequently non-representative of the general population of interest; they tend to 
+be sicker [@doi:10.1086/512821], more male [@doi:10.1016/j.neubiorev.2010.07.002],
```
Comment (agapow) [inline sections/03_categorize.md]: Get completely what you mean by "more", it reads a bit awkward. Maybe "those with illness, males and Europeans tend to be over-represented"

File: sections/03_categorize.md [hunk 7]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
-interpret, must remain cognizant of this possibility and make every effort to
-prevent harm from discriminatory predictions.
+important to consider algorithm interpretability alongside accuracy. A number of
+properties of genomic data will make this difficult.  First, research samples are 
+frequently non-representative of the general population of interest; they tend to 
+be sicker [@doi:10.1086/512821], more male [@doi:10.1016/j.neubiorev.2010.07.002], 
+and more European in ancestry [@doi:10.1371/journal.pbio.1001661]. One well-known 
+consequence of these biases in genomics is that penetrance is consistently lower 
+in the general population than would be implied by case-control data, as reviewed 
+in [@doi:10.1086/512821]. Moreover, genetic associations that hold in one population
```
Comment (agapow) [inline sections/03_categorize.md]: "that hold" -> "found"?

File: sections/03_categorize.md [hunk 8]
```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
...
+frequently non-representative of the general population of interest; they tend to 
+be sicker [@doi:10.1086/512821], more male [@doi:10.1016/j.neubiorev.2010.07.002], 
+and more European in ancestry [@doi:10.1371/journal.pbio.1001661]. One well-known 
+consequence of these biases in genomics is that penetrance is consistently lower 
+in the general population than would be implied by case-control data, as reviewed 
+in [@doi:10.1086/512821]. Moreover, genetic associations that hold in one population 
+may not hold in other populations with different patterns of linkage 
+disequilibrium (even when population stratification is explicitly controlled for; 
+[@doi:10.1038/nrg2813]). As a result, many genomic findings are of limited value for 
+people of non-European ancestry[@doi:10.1371/journal.pbio.1001661] and may lead
```
Comment (agapow) [inline sections/03_categorize.md]: "may" -> "may even"

**Comment by davharris** [comment]
Thanks folks! Let's see if this works better.

**Pull request merged**